apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -21,13 +21,14 @@ export const getFolders = () =>
     .execute()
 
 // NOTE: Base method for retrieving a resource - no distinction made on whether `blobId` exists
-const getById = (id: number) => db.selectFrom('Resource').where('id', '=', id)
+const getById = (id: number) =>
+  db.selectFrom('Resource').where('Resource.id', '=', id)
```

Comment:
bugfix - inner joins have 2 id cols so we need to be explicit which id we're referring to